### PR TITLE
Fixing top part of admin page

### PR DESCRIPTION
**Pull request by scottcwilson**
First block of fixes for #1914.

### CODE REVIEW

File: admin/attributes_controller.php [hunk 1]
```diff
@@ -699,10 +698,14 @@ function popupWindow(url) {
   window.open(url,'popupWindow','toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=no,resizable=yes,copyhistory=no,width=600,height=460,screenX=150,screenY=150,top=150,left=150')
 }
 </script>
+<?php
```
Comment (Zen4All) [inline admin/attributes_controller.php]: This puts the above JavaScript outside the html tags

Comment (scottcwilson) [reply admin/attributes_controller.php]: Update your branch and do a view source - it looks ok to me.

Comment (Zen4All) [reply admin/attributes_controller.php]: Is see the same errors in some of the files in #1935 where `require_once('includes/template/common/tplHtmlHeadLegacy.php');`
is put after the javascripts

**Comment by scottcwilson** [comment]
ARGH.  You are correct -I see what you mean now.

**Comment by Zen4All** [comment]
Maybe the scripts should be lowered to be inside body tag

**Comment by scottcwilson** [comment]
It's ok - I rearranged the inclusion of these files and it works now.  The trouble is, the example I used for how to do this was wrong, so I carried the error forward. :(  But the newly submitted files are fixed.

**Pull request merged**